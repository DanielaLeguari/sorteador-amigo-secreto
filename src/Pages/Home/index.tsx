import { Cabecalho } from "../../components/Cabecalho"
import { Card } from "../../components/Card"
import { Formulario } from "../../components/Formulario"
import { ListaParticipantes } from "../../components/ListaParticipantes.tsx"
import { Rodape } from "../../components/Rodape"


export const Home = () => {
    return (
        <>
            <Cabecalho />
            <Card>
                <section>
                    <h2>
                        Vamos come√ßar!
                    </h2>
                    <Formulario />
                    <ListaParticipantes />
                    <Rodape />
                </section>
            </Card>
        </>
    )
}