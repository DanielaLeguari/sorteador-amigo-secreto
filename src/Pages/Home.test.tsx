import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import { Home } from "./Home";

const mockNavegacao = jest.fn()

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavegacao
    }
})

describe("a pagina Home", () => {

    test("deve ser redenizada corretamente", () => {
        const { container } = render(<RecoilRoot>
            <Home />
        </RecoilRoot>);

        expect(container).toMatchSnapshot();
    })
})