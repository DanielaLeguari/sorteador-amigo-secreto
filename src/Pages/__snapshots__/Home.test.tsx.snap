// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina Home deve ser redenizada corretamente 1`] = `
<div>
  <header
    class="cabecalho"
  >
    <div
      aria-label="Logo do Sorteador"
      class="imagemLogo"
      role="img"
    />
    <img
      alt="Participante com um presente na mão"
      class="participante"
      src="/imagens/participante.png"
    />
  </header>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamos começar!
      </h2>
      <form>
        <div
          class="grupoInputBtn"
        >
          <input
            placeholder="Insira os nomes dos participantes"
            type="text"
            value=""
          />
          <button
            disabled=""
          >
            Adicionar
          </button>
        </div>
        
      </form>
      <ul />
      <footer
        class="rodapeConfiguracoes"
      >
        <button
          class="botao"
          disabled=""
        >
          Iniciar brincadeira
        </button>
        <img
          alt="Sacolas de compras"
          src="/imagens/sacolas.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
